appeal/job_handler.go

```diff
@@ -7,16 +7,16 @@ import (
 	"time"
 
 	"github.com/odpf/guardian/domain"
-	"go.uber.org/zap"
+	saltLog "github.com/odpf/salt/log"
```

Comment:
@wisdommatt Let's avoid alias here as well.

Reply:
The reason for the alias is because that particular file imports the `log` std package

What should i do in this situation ?

Reply:
@wisdommatt I think we can replace those logs from `log` package with the salt's one

Reply:
> @wisdommatt I think we can replace those logs from `log` package with the salt's one

Noted

Reply:
I just pushed a commit to resolve this